pkg/ipcache/types.go

```diff
@@ -15,13 +18,36 @@ import (
 // to generate ipcache output based on a range of inputs.
 type prefixInfo map[types.ResourceID]*resourceInfo
 
+func (p prefixInfo) logConflicts(scopedLog *logrus.Entry) {
+	var (
+		overrideID         types.IdentityOverride
+		overrideResourceID types.ResourceID
+	)
+	for resourceID, v := range p {
+		if v.identityOverride.IsValid() {
+			if overrideID.IsValid() {
+				scopedLog.WithFields(logrus.Fields{
+					logfields.Identity:            overrideID,
+					logfields.Resource:            overrideResourceID,
+					logfields.ConflictingIdentity: v.identityOverride,
+					logfields.ConflictingResource: resourceID,
+				}).Warning("Detected conflicting identity override for prefix")
```

Comment:
Suggestion for a bit more clear user impact:

```suggestion
				}).Warning("Detected conflicting identity override for prefix. This may cause connectivity issues for this address.")
```

It'd be nice to include the prefix in the logs here too, to assist the user in understanding the potential impact.